more_itertools/more.py

```diff
@@ -1106,3 +1108,83 @@ def always_iterable(obj):
         return obj,
 
     return obj
+
+def adjacent(predicate, iterable, distance=1):
+    """
+    Returns an iterable over ``(bool, item)`` tuples where the ``item`` is drawn
```

Comment:
Nitpicks: I think Erik prefers no line break after `"""` for docstrings, and [PEP 257](https://www.python.org/dev/peps/pep-0257/#id16) mystifyingly wants things in the imperative mood ("Return..." and not "Returns...")